### DISCUSSION TITLE
framewiki: Contributing

### DISCUSSION THREAD

**Discussion by giscus** [framewiki/framewiki.net#29] [Talk]
# Framewiki: Contributing

The unofficial community-driven Framework wiki

https://framewiki.net/framewiki:contributing

**Comment by KellyNyanbinary** [comment]
@ Morpheus636 are citations really supposed to be in the “order in which the source was added”? It looks weird with the random order that resulted from how I wrote the specs.

**Reply by Morpheus636** [reply]
It's possible that language may need clarification. Once a source is added and *deployed*, it's number shouldn't change, even if its order on the page changes.

**Reply by Morpheus636** [reply]
It looks like Jekyll actually puts them in order for you when you use footnote formatting, so I'll actually remove that guidance. The concern was having to go through and manually re-arrange source numbers.

**Reply by KellyNyanbinary** [reply]
That's amazing. Thank you.
